docs.md

```diff
@@ -26,6 +26,8 @@ These include:
   Kubernetes Service on Azure Stack HCI - quick way to get started hosting
   Windows and Linux containers in your data center.
 
+  Customers external to Microsoft can use CBL-Mariner via [AKS as a node OS](https://aka.ms/azure-linux-docs). 
```

Comment:
```suggestion
Customers external to Microsoft can use CBL-Mariner via [AKS as a node OS](https://aka.ms/azure-linux-docs). 
```

Indentation looks odd in the actual rendered Markdown, seemingly attaching this statement to AKS-HCI even though it's a global statement.